sympy/printing/mathematica.py

```diff
@@ -107,7 +107,6 @@
     "DiracDelta": [(lambda x: True, "DiracDelta")],
     "Heaviside": [(lambda x: True, "HeavisideTheta")],
     "KroneckerDelta": [(lambda *x: True, "KroneckerDelta")],
-    "LambertW": [(lambda x: True, "ProductLog")],
```

Comment:
I do not remember, but does it work to put `*x` there or is there an issue if it may be one or several arguments?

Reply:
Ahh, they should be in reversed order. Never mind.

Reply:
Yes, I think that the original design turned out to have some limitations like this, and I didn't want to add more complicated options to `_print_Function` only to give a special treatment for this.